SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -0,0 +1,51 @@
+
+You need to add two environment variables: one to set the path to the Unreal Engine fork directory, and another one to set the path to the Linux cross-platform support.
+
+1. Go to **Control Panel > System and Security > System > Advanced system settings > Advanced > Environment variables**.
+1. Create a system variable named **UNREAL_HOME**.
+1. Set the variable value to be the path to the directory you cloned the Unreal Engine fork into.
+1. Make sure that the new environment variable is registered by restarting your terminal and running `echo %UNREAL_HOME%` (Command Prompt) or `echo $Env:UNREAL_HOME` (PowerShell). If the environment variable is registered correctly, this returns the path to the directory you cloned the Unreal Engine fork into. If it doesn’t, check that you’ve set the environment variable correctly.
+1. Create a system variable named **LINUX_MULTIARCH_ROOT**.
+1. Set the variable value to be the path to the directory of your unzipped Linux cross compilation toolchain.
+1. Make sure that the new environment variable is registered by restarting your terminal and running `echo %LINUX_MULTIARCH_ROOT%` (Command Prompt) or `echo $Env:LINUX_MULTIARCH_ROOT` (PowerShell).
+If the environment variable is registered correctly, this returns the path you unzipped `v11_clang-5.0.0-centos7.zip` into. If it doesn’t, check that you’ve set the environment variable correctly.
+
+### Building Unreal Engine
+
+> Building the Unreal Engine fork from source could take up to a few hours.
+
+1. Open **File Explorer** and navigate to the directory you cloned the Unreal Engine fork into.
+1. Double-click **`Setup.bat`**.
+</br>This installs prerequisites for building Unreal Engine 4, and may take a while.
+    >  While running the Setup file, you should see `Checking dependencies (excluding Mac, Android)...`. If it also says `excluding Linux`, make sure that you set the environment variable `LINUX_MULTIARCH_ROOT` correctly, and run the Setup file again.
+1. In the same directory, double-click **`GenerateProjectFiles.bat`**.
+</br>This sets up the project files required to build Unreal Engine 4.
+
+    > Note: If you encounter an `error MSB4036: The "GetReferenceNearestTargetFrameworkTask" task was not found` when building with Visual Studio 2017, check that you have NuGet Package Manager installed via the Visual Studio installer.
+1. In the same directory, open **UE4.sln** in Visual Studio.
+1. In Visual Studio, on the toolbar, go to **Build** > **Configuration Manager** and set your active solution configuration to **Development Editor** and your active solution platform to **Win64**.
+1. In the Solution Explorer window, right-click on the **UE4** project and select **Build** (you may be prompted to install some dependencies first).</br>
+    This builds Unreal Engine, which can take up to a couple of hours.
+1. Once the build succeeds, in the Solution Explorer, find **Programs** > **AutomationTool**. Right-click this project and select **Build**.
```

Comment:
This step should not be necessary. cc: @jared-improbable

Reply:
removed